cityofaustin/atd-moped
# Reworks work activities in the project list view

## Associated issues

## Testing
**URL to test:**  Local

**Steps to test:**
1. Use the editor add a few work activities to projects with multiple task orders.
2. Make sure the list view loads normally
3. these two statements should have the same result:
- `select count(project_id) from project_list_view;`
- `select count(project_id) from moped_project where is_deleted = false;`

---
#### Ship list
